ehrapy/data/_datasets.py

```diff
@@ -181,3 +181,237 @@ def chronic_kidney_disease(
+) -> AnnData:
+    """Loads the Echocardiogram Data Set
+
+    More details: http://archive.ics.uci.edu/ml/datasets/Echocardiogram
+    Preprocessing: https://github.com/theislab/ehrapy-datasets/blob/thyroid_dataset/echocardiogram/echocardiogram.ipynb
+
+    Args:
+        columns_obs_only: Columns to include in obs only and not X.
+
+    Returns:
+        :class:`~anndata.AnnData` object of the Echocardiogram Data Set
+
+    Example:
+        .. code-block:: python
+
+            import ehrapy as ep
+
+            adata = ep.dt.echocardiogram()
+    """
+    adata = read(
+        dataset_path=f"{ehrapy_settings.datasetdir}/echocardiogram.csv",
+        download_dataset_name="echocardiogram.csv",
+        backup_url="https://figshare.com/ndownloader/files/34179306",
+        columns_obs_only=columns_obs_only,
+        extension="csv",
+        index_column="patient_id",
+    )
+
+    return adata
+
+
+def hepatitis(
+    columns_obs_only: dict[str, list[str]] | list[str] | None = None,
+) -> AnnData:
+    """Loads the Hepatitis Data Set
+
+    More details: http://archive.ics.uci.edu/ml/datasets/Hepatitis
+    Preprocessing: https://github.com/theislab/ehrapy-datasets/blob/thyroid_dataset/hepatitis/hepatitis.ipynb
+
+    Args:
+        columns_obs_only: Columns to include in obs only and not X.
+
+    Returns:
+        :class:`~anndata.AnnData` object of the Hepatitis Data Set
+
+    Example:
+        .. code-block:: python
+
+            import ehrapy as ep
+
+            adata = ep.dt.hepatitis()
+    """
+    adata = read(
+        dataset_path=f"{ehrapy_settings.datasetdir}/hepatitis.csv",
+        download_dataset_name="hepatitis.csv",
+        backup_url="https://figshare.com/ndownloader/files/34179318",
+        columns_obs_only=columns_obs_only,
+        extension="csv",
+        index_column="patient_id",
+    )
+
+    return adata
+
+
+def statlog_heart(
+    columns_obs_only: dict[str, list[str]] | list[str] | None = None,
+) -> AnnData:
+    """Loads the Statlog (Heart) Data Set
+
+    More details: http://archive.ics.uci.edu/ml/datasets/Statlog+%28Heart%29
+    Preprocessing: https://github.com/theislab/ehrapy-datasets/blob/main/statlog_heart/statlog_heart.ipynb
+
+    Args:
+        columns_obs_only: Columns to include in obs only and not X.
+
+    Returns:
+        :class:`~anndata.AnnData` object of the Statlog (Heart) Data Set
+
+    Example:
+        .. code-block:: python
+
+            import ehrapy as ep
+
+            adata = ep.dt.statlog_heart()
+    """
+    adata = read(
+        dataset_path=f"{ehrapy_settings.datasetdir}/statlog_heart.csv",
+        download_dataset_name="statlog_heart.csv",
+        backup_url="https://figshare.com/ndownloader/files/34179327",
+        columns_obs_only=columns_obs_only,
+        extension="csv",
+        index_column="patient_id",
+    )
+
+    return adata
+
+
+def thyroid(name: str, encoded: bool = False) -> AnnData:
```

Comment:
Oh, wait. I think this should always return the full dataset by default. Not a subset.

Reply:
No problem, already fixed.